src/components/tree-view/tree-view.css

```diff
@@ -0,0 +1,17 @@
+.tree-view {
```

Comment:
i put the styling inside the component css and not the demo because they are probably the minimal thing for the component to actually look like a tree